app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PagesController < ApplicationController
+  def show
+    render template: "pages/#{params[:page]}"
```

Comment:
Let's make it Rails way: create an empty `about` method and update routes for it.